knimin/tests/test_ag_search.py

```diff
@@ -64,7 +65,9 @@ def test_post_barcode(self):
                              field == 'ag_kit_barcode_id'):
                             self.assertNotIn(sample[field], response.body)
                         else:
-                            self.assertIn(sample[field], response.body)
+                            exp = xhtml_escape_recursive(sample[field])
```

Comment:
because it depends on the values in the DB which are not guaranteed to stay the same when we switch to Jose's new dump